specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.3/TextAnalytics.json

```diff
@@ -1947,7 +1947,8 @@
             "failed",
             "cancelled",
             "cancelling",
-            "partiallyCompleted"
+            "partiallyCompleted",
```

Comment:
what is the different meaning of `partiallyCompleted` and` partiallySucceeded`?

Reply:
if I will have to guess, I think the service uses `partiallySucceeded` instead of `partiallyCompleted`. cc @kayousef

Reply:
@omaremadeldin can we rollout a fix for this real quick?

Reply:
@kayousef What is the status of this?